BWA_CHIP/submit_BWA_CHIP.sh

```diff
@@ -26,11 +49,32 @@ else
     array_length=$(wc -l < ${fastq_list}) #get the number of FASTQs 
 
     sbatch -o "${output_directory}/Logs/%A_%a.log" `#put into log` \
-        -a "1-${array_length}" `#initiate job array equal to the number of fastq files` \
-        "${code_directory}/BWA_CHIP.sh" \
+            -a "1-${array_length}" `#initiate job array equal to the number of fastq files` \
+            -W `#indicates to the script not to move on until the sbatch operation is complete` \
+            "${code_directory}/inif2.sh" \
             ${parent_directory} \
             ${output_directory} \
             ${min_coverage} \
             ${min_var_freq} \
-            ${p_value}
+            ${p_value} \
+            ${get_mutect} \
+            ${get_varscan} \
+            ${get_haplotype}
+fi
+
+wait
+
```

Comment:
I forgot that we need to add `module load R/4.0` before calling `Rscript`.  We could do it once up here, or inside each `if` statement.